app/Http/Controllers/Api/V1/TeamController.php

```diff
@@ -574,12 +593,13 @@ public function addMembersToProject(TeamAddMemberRequest $addMemberRequest, Team
         $data = $addMemberRequest->validated();
         $assigned_members = [];
         $suborganization = $team->organization;
-       
+
         foreach ($data['ids'] as $member_id) {
             $exist_user_id = $suborganization->users()->where('user_id', $member_id)->first();
             if (!$exist_user_id) {
                 return response([
-                    'errors' => ['All selected members must be added in ' . $suborganization->name . ' organization first.'],
+                    'errors' =>
```

Comment:
Please revert it back to previous, if it makes horizontal scroll then only we use break line